domain/src/main/java/life/qbic/domain/user/Email.java

```diff
@@ -0,0 +1,69 @@
+package life.qbic.domain.user;
+
+import java.util.Objects;
+import life.qbic.domain.usermanagement.policies.EmailFormatPolicy;
+import life.qbic.domain.usermanagement.policies.PolicyCheckReport;
+import life.qbic.domain.usermanagement.policies.PolicyStatus;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public class Email {
```

Comment:
Why not `EmailAddress` directly? The user does not have an email attached to it but an email address.